addons/account/tests/test_invoice_taxes.py

```diff
@@ -715,3 +715,25 @@ def test_tax_calculation_multi_currency(self):
         self.assertRecordValues(invoice.line_ids.filtered(lambda l: l.account_id.account_type == 'asset_receivable'), [{
             'balance': 686.54,
         }])
+
+    def test_tax_account_receivable(self):
+        self.company_data['default_account_receivable'].write({
+            'tax_ids': [Command.set(self.percent_tax_1.ids)]
+        })
```

Comment:
Why would anybody do that?

Reply:
I guess to have the tax line added automatically when needed (for example when creating a journal entry).

New case: opw-3348999